init-tools.sh

```diff
@@ -8,7 +8,7 @@ initDistroName()
             export __DistroRid=""
         else
             source /etc/os-release
-            export __DistroRid="$ID.$VERSION_ID-$__BuildArch"
+            export __DistroRid="$ID.$VERSION_ID-x64"
```

Comment:
It would have never worked there before, we don't have ARM based CLIs, so we could have never restored a tool run-time for arm anyway.  This tool stuff runs on the host anyway, and ARM just cross builds today, right?